### PR TITLE
Kafka bulk support documentation

### LINKED ISSUE #2825: Kafka bulk support documentation
**What content needs to be created or modified?**

TODO: add more details, to be triaged for v1.9

**Describe the solution you'd like**

**Where should the new material be placed?**

**The associated pull request from dapr/dapr, dapr/components-contrib, or other Dapr code repos**

**Additional context**

### PR DESCRIPTION
**Pull request by mukundansundar**
Signed-off-by: Mukundan Sundararajan <65565396+mukundansundar@users.noreply.github.com>

Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**
- [ ] Commits are signed with Developer Certificate of Origin (DCO - [learn more](https://docs.dapr.io/contributing/contributing-overview/#developer-certificate-of-origin-signing-your-work))
- [ ] [Read the contribution guide](https://docs.dapr.io/contributing/docs-contrib/contributing-docs/)
- [ ] Commands include options for Linux, MacOS, and Windows within codetabs
- [ ] New file and folder names are globally unique
- [ ] Page references use shortcodes instead of markdown or URL links
- [ ] Images use HTML style and have alternative text
- [ ] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Added configuration parameter for bulk subscribe. 
No configuration parameters needed for bulk publish as of now. 

WIP since there is a link needed to be added see TODO in the doc.

## Issue reference

closes #2825

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md [hunk 1]
```diff
@@ -293,6 +293,19 @@ auth:
   secretStore: <SECRET_STORE_NAME>
 ```
 
+## Sending and receiving multiple messages
+
+Apache Kafka component natively supports sending and receiving multiple messages in a single operation (Bulk Pubsub feature).
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md]:
```suggestion
Apache Kafka component supports sending and receiving multiple messages in a single operation using the bulk Pub/sub API.
```

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md [hunk 2]
```diff
@@ -293,6 +293,19 @@ auth:
...
+
+### Configuring bulk subscribe
+
+When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. (TODO: Add link to bulk subscribe docs)
+
+| Configuration | Default |
+|----------|---------|
+| `maxBulkAwaitDurationMs` | `10000` (10s) |
```
Comment (msfussell) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md:306]: Same comment as the Azure Service bus docs pR #3054 

These fields needs to be added to the component spec example at the start as well as into the table of values. Putting them here does not make them discoverable. The table is the source of truth for the spec.

Comment (mukundansundar) [reply daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md:306]: @msfussell This is not a component specific spec. This is a bulk Subscribe specific spec. 
That is why I added a TBD. This has to be changed after https://github.com/dapr/docs/pull/3077 PR is merged. Example config in that PR https://github.com/dapr/docs/pull/3077/files#diff-081f2bf4c37527e63a2ebbfc0ca6b08ee9606a6650d84c54dc2b374d61dd85e2R170-R186

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md [hunk 3]
```diff
@@ -293,6 +293,19 @@ auth:
   secretStore: <SECRET_STORE_NAME>
 ```
 
+## Sending and receiving multiple messages
+
+Apache Kafka component natively supports sending and receiving multiple messages in a single operation (Bulk Pubsub feature).
+
+### Configuring bulk subscribe
+
+When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. (TODO: Add link to bulk subscribe docs)
```
Comment (hhunter-ms) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md]:
```suggestion
When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. Learn more about [the bulk subscribe API]({{< ref pubsub-bulk.md >}}).
```

**Comment by hhunter-ms** [comment]
@mukundansundar friendly bump to address reviews

**Comment by mukundansundar** [comment]
> @mukundansundar friendly bump to address reviews

Done.

**Review by hhunter-ms — APPROVED** [review]
lgtm

File: daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md [hunk 4]
```diff
@@ -293,6 +293,19 @@ auth:
   secretStore: <SECRET_STORE_NAME>
 ```
 
+## Sending and receiving multiple messages
+
+Apache Kafka component supports sending and receiving multiple messages in a single operation using the bulk Pub/sub API.
+
+### Configuring bulk subscribe
+
+When subscribing to a topic, you can configure `bulkSubscribe` options. Refer to [Subscription methods]({{< ref subscription-methods >}}) for more details. Learn more about [the bulk subscribe API]({{< ref pubsub-bulk.md >}}).
```
Comment (shubham1172) [inline daprdocs/content/en/reference/components-reference/supported-pubsub/setup-apache-kafka.md:302]: @mukundansundar will  "ref subscription-methods" contain the bulk sub docs?

**Pull request merged**